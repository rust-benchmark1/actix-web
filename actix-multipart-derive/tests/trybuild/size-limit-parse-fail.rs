use actix_multipart::form::{text::Text, MultipartForm};

#[derive(MultipartForm)]
struct Form {
    #[multipart(limit = "2 bytes")]
    description: Text<String>,
}

#[derive(MultipartForm)]
struct Form2 {
    #[multipart(limit = "2 megabytes")]
    description: Text<String>,
}

#[derive(MultipartForm)]
struct Form3 {
    #[multipart(limit = "four meters")]
    description: Text<String>,
}

fn main() {}
