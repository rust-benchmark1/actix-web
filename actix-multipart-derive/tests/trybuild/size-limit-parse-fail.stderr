error: Could not parse size limit `2 bytes`: invalid digit found in string
 --> tests/trybuild/size-limit-parse-fail.rs:6:5
  |
6 |     description: Text<String>,
  |     ^^^^^^^^^^^

error: Could not parse size limit `2 megabytes`: invalid digit found in string
  --> tests/trybuild/size-limit-parse-fail.rs:12:5
   |
12 |     description: Text<String>,
   |     ^^^^^^^^^^^

error: Could not parse size limit `four meters`: invalid digit found in string
  --> tests/trybuild/size-limit-parse-fail.rs:18:5
   |
18 |     description: Text<String>,
   |     ^^^^^^^^^^^
